src/openklant/components/klantinteracties/models.py

```diff
@@ -0,0 +1,340 @@
+import uuid
+
+from django.core.validators import MinLengthValidator, validate_integer
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from vng_api_common.descriptors import GegevensGroepType
+
+from .constants import SoortPartij
+
+
+class Partij(models.Model):
+    id = models.UUIDField(
+        primary_key=True,
+        unique=True,
+        default=uuid.uuid4,
+        help_text=_("Unieke (technische) identificatiecode van de partij."),
+    )
+    nummer = models.CharField(
+        _("Nummer"),
+        help_text=_(
+            "Uniek identificerend nummer dat tijdens communicatie tussen mensen kan "
+            "worden gebruikt om de specifieke partij aan te duiden."
+        ),
+        validators=[validate_integer],
+        max_length=10,
+    )
+    interne_notitie = models.CharField(
+        _("Interne notitie"),
+        help_text=_(
+            "Mededelingen, aantekeningen of bijzonderheden over de partij, bedoeld voor intern gebruik."
+        ),
+        max_length=1000,
+        blank=True,
+    )
+    soort_partij = models.CharField(
+        _("Soort partij"),
+        help_text=_("Geeft aan van welke specifieke soort partij sprake is."),
+        max_length=14,
+        choices=SoortPartij.choices,
+    )
+    indicatie_geheimhouding = models.BooleanField(
+        _("Indicatie geheimhouding"),
+        help_text=_(
+            "Geeft aan of de verstrekker van partijgegevens heeft aangegeven dat "
+            "deze gegevens als geheim beschouwd moeten worden."
+        ),
+    )
+    voorkeurskanaal = models.CharField(
+        _("Voorkeurskanaal"),
+        help_text=_(
+            "Kanaal dat de partij bij voorkeur gebruikt voor contact met de gemeente."
+        ),
+        max_length=50,
+        blank=True,
+    )
+    voorkeurstaal = models.CharField(
```

Comment:
Wijzigen naar:

max_length=3
helptekst:
Taal, in ISO 639-2/B formaat, waarin de partij bij voorkeur contact heeft met de gemeente. Voorbeeld: nld. Zie: https://www.iso.org/standard/4767.html